### ISSUE TITLE
Create content in the admin UI

### ISSUE THREAD

**Issue by JanLenoch** [Kentico/xperience-training-13#28]
### Who

- [ ] Anonymous visitor
- [ ] Patient
- [ ] Doctor
- [x] Content editor
- [ ] Marketer
- [ ] Administrator
- [ ] Developer

### Needs what

Content types, pages, and other data that's aligned to the content tree-based routing.

### Why

Why is the feature needed?

### Issue expands upon

This issue expands upon another issue(s): #8 

### Blockers

This issue is blocked by the following issue(s): #`issue number`
